### PR TITLE
Remembering Fire Center

### PR DESCRIPTION
**Pull request by JulianForeman**
# Test Links:
[Percentile Calculator](https://wps-pr-1498.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-1498.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-1498.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-1498.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-1498.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=1108&f=c5&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN#state=2ec784ca-c46a-49d0-b2b3-1cf32a9015a2&session_state=7d9447c8-db66-4661-b4cb-03d2ac0d1d8f&code=32292df4-2bdf-4f90-a4a8-c8dbcda682a9.7d9447c8-db66-4661-b4cb-03d2ac0d1d8f.2b63f390-f3dc-43ae-89f2-016453863476)
[Fire Behaviour Advisory](https://wps-pr-1498.apps.silver.devops.gov.bc.ca/fire-behaviour-advisory)
[HFI Calculator](https://wps-pr-1498.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: web/cypress/integration/fba-map-page.spec.ts [hunk 1]
```diff
@@ -17,4 +13,15 @@ describe('Fire Behaviour Advisory Page', () => {
...
   })
+
+  it('Sets the fireCenter in local storage when it is changed in dropdown', () => {
+    cy.wait('@getStations')
+    cy.getByTestId('fire-center-dropdown')
+      .click()
+      .type('{downArrow}')
+      .type('{enter}')
+      .should(() => {
+        expect(localStorage.getItem('preferredFireCenter')).to.equal('50')
```
Comment (conbrad) [inline web/cypress/integration/fba-map-page.spec.ts:24]: Did not know cypress has access to `localStorage` right off the bat like this. Cool! Does this test fail if you change the equal assertion to another value?

File: web/package.json [hunk 2]
```diff
@@ -29,8 +29,8 @@
     "@types/react-redux": "^7.1.16",
     "@types/react-router-dom": "^5.3.0",
     "@types/webpack-env": "^1.15.1",
-    "@typescript-eslint/parser": "^5.0.0",
     "@typescript-eslint/eslint-plugin": "^5.0.0",
+    "@typescript-eslint/parser": "^5.0.0",
```
Comment (conbrad) [inline web/package.json:33]: Why this change?

**Review by conbrad — CHANGES_REQUESTED** [review]
Nice. One other thing is we shouldn't be committing the giant `package-lock.json` since we already have a `yarn.lock` for this. It may be that an `npm install` was run on your machine instead of a `yarn install`.

Let's delete the `package-lock.json` file.

File: web/package.json (hunk 2 above)
Comment (JulianForeman) [reply web/package.json:33]: This was most likely an accident when I was trying to use packages and deleted/reinstalled my node modules. I'm not sure if this was an actual version change or if the space it was in is just different.

File: web/cypress/integration/fba-map-page.spec.ts (hunk 1 above)
Comment (JulianForeman) [reply web/cypress/integration/fba-map-page.spec.ts:24]: Yes because right now it's expecting the local storage to be the id of the first item in the dropdown when selected.

File: web/package.json (hunk 2 above)
Comment (conbrad) [reply web/package.json:33]: Easy to undo with `git checkout origin/main -- path/to/file`

File: web/cypress/integration/fba-map-page.spec.ts (hunk 1 above)
Comment (Sybrand) [reply web/cypress/integration/fba-map-page.spec.ts:24]: Phew - I was worried this was going to be an issue. Have tripped up on that before.

**Review by Sybrand — CHANGES_REQUESTED** [review]
Same comment as Connor - pls. get rid of package-lock.json

File: web/package.json (hunk 2 above)
Comment (conbrad) [reply web/package.json:33]: @JulianForeman please fix this above ^

**Review by Sybrand — APPROVED** [review]

**Review by conbrad — APPROVED** [review]

**Review by andrea-williams — APPROVED** [review]
